### ISSUE TITLE
You cannot replace newlines in a list object.

### ISSUE THREAD

**Issue by 133794m3r** [lockedbyte/cryptovenom#6]
<pre>
  ____                  _     __     __                         
 / ___|_ __ _   _ _ __ | |_ __\ \   / /__ _ __   ___  _ __ ___  
| |   | '__| | | | '_ \| __/ _ \ \ / / _ \ '_ \ / _ \| '_ ` _ \ 
| |___| |  | |_| | |_) | || (_) \ V /  __/ | | | (_) | | | | | |
 \____|_|   \__, | .__/ \__\___/ \_/ \___|_| |_|\___/|_| |_| |_|
            |___/|_|                                            
            
Version: 1.0.1       << The Cryptography Swiss Army knife >>
-------------------------------------------------------------------

-[MENU]-
        
    0) Crackssistant (NOT YET)
    1) Symmetric Algorithms
    2) Asymmetric Algorithms
    3) Encoding Algorithms
    4) Classical Algorithms
    5) Hash Algorithms
    6) Mathematical Functions
    7) Algorithm Identifiers (Not YET)
    8) String Manipulation
    9) Randomization Functions
   10) Other
   11) About CryptoVenom
    
   99) Exit
   

[=] Option: 4

-[MENU:Classical]-

    1) Caesar
    2) Vigenere Cipher
    3) Playfair Cipher
    4) Polybius Square
    5) Morse
    6) Atbash
    7) Baconian
    8) AutoKey
    9) Beaufort Cipher
   10) Railfence Cipher
   11) Simple Substitution Cipher
   12) Columnar Tramposition
   13) Bifid
   14) Foursquare Cipher
   15) Fractionated Morse
   16) Enigma
   17) Gronsfeld Cipher
   18) Porta
   19) RunningKey
   20) ADFGVX Cipher
   21) ADFGX Cipher
   22) Affine Cipher
   23) Vernam Cipher

   99) Exit
    

[=] Option: 11

-=[OPTIONS]=-

   1) Encrypt
   2) Decrypt
   3) Cracker
   4) Brute Force
   

[=] Option: 4
[=] [F]ile or [T]ext: f
[=] Input file path: sub
[=] Wordlist Path: american-english
Traceback (most recent call last):
  File "modules/classical/substitution/menu.py", line 172, in <module>
    SimpleSubstitutionBF(text, dic)
  File "/home/macarthur/Documents/ctfs/cryptovenom-master/modules/classical/substitution/bruteforce.py", line 31, in SimpleSubstitutionBF
    keys = keys.replace('\n', '')
AttributeError: 'list' object has no attribute 'replace'
</pre>
I'm beyond n00b level with python so I don't know the proper way to modify it properly. I tried using a lamda and mapping the output and replacing it but that doesn't work either so I'm stumped as to the fix.

**Comment by lockedbyte** [comment]
Hi,

Thanks @133794m3r for your report!!

I added a fix code.

I'll close the issue when you confirm me it's solved or not!

**Comment by 133794m3r** [comment]
I'll test it tomorrow. Sorry for no remembering to check this again. I've been busy on HackTheBox breaking boxes and writing python solvers since I've been teaching myself python and been trying to find reasons to use it.

**Comment by 133794m3r** [comment]
By the way would you be open to a pull request to add a Navajo Encoder/Decoder to your system. I can add it to the classical ciphers menu. My current implementation doesn't deal with word collisions(when the same navajo word is used both for the alphabet and also for an actual code word) nor does it deal well with compound words. But it can easily be neutered to be a simple alphabetical encoder/decoder that'll work just as well as dcodes and in reality better since it's local and it'd be licensed under the GPL.

Menu entry would be.
Classical->Navajo

- Encoder
- Decoder 
- Exit

Basically something like that. Since it's not an encoding per-se as the chosen word for the alphabet is chosen randomly during encoding.

**Comment by 133794m3r** [comment]
```
-[MENU]-
        
    0) Crackssistant (NOT YET)
    1) Symmetric Algorithms
    2) Asymmetric Algorithms
    3) Encoding Algorithms
    4) Classical Algorithms
    5) Hash Algorithms
    6) Mathematical Functions
    7) Algorithm Identifiers (Not YET)
    8) String Manipulation
    9) Randomization Functions
   10) Other
   11) About CryptoVenom
    
   99) Exit
   

[=] Option: 4

-[MENU:Classical]-

    1) Caesar
    2) Vigenere Cipher
    3) Playfair Cipher
    4) Polybius Square
    5) Morse
    6) Atbash
    7) Baconian
    8) AutoKey
    9) Beaufort Cipher
   10) Railfence Cipher
   11) Simple Substitution Cipher
   12) Columnar Tramposition
   13) Bifid
   14) Foursquare Cipher
   15) Fractionated Morse
   16) Enigma
   17) Gronsfeld Cipher
   18) Porta
   19) RunningKey
   20) ADFGVX Cipher
   21) ADFGX Cipher
   22) Affine Cipher
   23) Vernam Cipher

   99) Exit
    

[=] Option: 11


-=[OPTIONS]=-

   1) Encrypt
   2) Decrypt
   3) Cracker
   4) Brute Force
   

[=] Option: 4
[=] [F]ile or [T]ext: t
[=] Text: fbzrguvat
[=] Wordlist Path: /usr/share/dict/american-english
Traceback (most recent call last):
  File "modules/classical/substitution/menu.py", line 172, in <module>
    SimpleSubstitutionBF(text, dic)
  File "/home/macarthur/Documents/ctfs/cryptovenom-master/modules/classical/substitution/bruteforce.py", line 34, in SimpleSubstitutionBF
    out = SimpleSubstitution(i).decipher(ctext)
  File "/home/macarthur/.local/lib/python2.7/site-packages/pycipher/simplesubstitution.py", line 23, in __init__
    assert len(key) == 26
AssertionError
```

it still isn't working. I guess it's some external dependency this time. I did plain-ol ceaser to test it as I don't have my original text i was trying to bruteforce anymore. I figured that if it would work with ceaser(literal rot13) then it'd work for anything.
